README.md

```diff
@@ -413,6 +414,68 @@ Tester: do_forward_flips(forward_soberness_check=18, num_flips=50)
+
+In addition, it can also decode a hex request and display its parameters 
+mapped to a service. 
+
+```
+$ find $BASE_DIR/odxtools/examples/somersault.pdx -D 10 00
+
+
+=====================================
+somersault_lazy, somersault_assiduous
+=====================================
+
+
+ session_start <ID: somersault.service.session_start>
+  Message format of a request:
+           7     6     5     4     3     2     1     0  
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+      0 | sid (8 bits)                                  |
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+      1 | id (8 bits)                                   |
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+   Parameter(short_name='sid', type='CODED-CONST', semantic=None, byte_position=0, bit_length=8, coded_value='0x10')
+   Parameter(short_name='id', type='CODED-CONST', semantic=None, byte_position=1, bit_length=8, coded_value='0x0')
+  Number of positive responses: 1
+  Message format of a positive response:
+           7     6     5     4     3     2     1     0  
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+      0 | sid (8 bits)                                  |
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+      1 | can_do_backward_flips (8 bits)                |
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+   Parameter(short_name='sid', type='CODED-CONST', semantic=None, byte_position=0, bit_length=8, coded_value='0x50')
+   Parameter(short_name='can_do_backward_flips', type='VALUE', semantic=None, byte_position=1, bit_length=8, dop_ref='somersault.DOP.boolean')
+    DataObjectProperty('boolean', category='TEXTTABLE', internal_type='A_UINT32', physical_type='A_UNICODE2STRING')
+  Number of negative responses: 1
+  Message format of a negative response:
+           7     6     5     4     3     2     1     0  
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+      0 | sid (8 bits)                                  |
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+      1 | rq_sid (8 bits)                               |
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+      2 | response_code (8 bits)                        |
+        +-----+-----+-----+-----+-----+-----+-----+-----+
+   Parameter(short_name='sid', type='CODED-CONST', semantic=None, byte_position=0, bit_length=8, coded_value='0x7f')
+   Parameter(short_name='rq_sid', type='MATCHING-REQUEST-PARAM', semantic=None, byte_position=1)
+    Request byte position = 0, byte length = 1
+   Parameter(short_name='response_code', type='VALUE', semantic=None, byte_position=2, bit_length=8, dop_ref='somersault.DOP.error_code')
+    DataObjectProperty('error_code', category='IDENTICAL', internal_type='A_UINT32', physical_type='A_UINT32')
+
+Decoded Request('start_session'):
+	sid: 16
+	id: 0
+```
+
+`find $BASE_DIR/odxtools/examples/somersault.pdx -d 10 00` would display the same information, 
```

Comment:
s/find/odxtools find/ (`find` is a standard unix command). Sorry that I missed this in the review...

Reply:
No worries, missed it, after copying and pasting the command - fixed and repushed.